firebase-sessions/src/main/kotlin/com/google/firebase/sessions/FirebaseSessions.kt

```diff
@@ -46,18 +45,34 @@ internal constructor(
       blockingDispatcher,
       backgroundDispatcher,
       firebaseInstallations,
-      applicationInfo
+      applicationInfo,
```

Comment:
That's a style thing. I like the trailing `,` because when we add another line, we don't have to also edit the previous last line. Kotlin style encourages it https://kotlinlang.org/docs/coding-conventions.html#trailing-commas